best-practices/README.md

```diff
@@ -187,6 +187,14 @@ Email
 [MailView]: https://github.com/37signals/mail_view
 [ActionMailer Preview]: http://api.rubyonrails.org/v4.1.0/classes/ActionMailer/Base.html#class-ActionMailer::Base-label-Previewing+emails
 
+Web
+---
+
+* Prefer local assets to externally linked assets.
```

Comment:
For http/2, we should prefer local assets instead of a CDN. But we're not close to http/2 yet.